docs/runtimes/spark-operator.ipynb

```diff
@@ -73,11 +73,8 @@
     "# set spark executor volume mount\n",
     "# sj.function.with_executor_host_path_volume(\"/host/path\", \"/mount/path\")\n",
     "\n",
-    "# confs are also supported\n",
-    "sj.spec.spark_conf[\"spark.eventLog.enabled\"] = True\n",
```

Comment:
This currently fails with 
```
java.io.FileNotFoundException: File file:/tmp/spark-events does not exist
```
A fix for this issue will likely be in spark-app base image and not in mlrun.